internal/worker/server_test.go

```diff
@@ -172,6 +172,39 @@ func TestUpdate(t *testing.T) {
 	test.TestRoute(t, handler, false, "PATCH", fmt.Sprintf("/api/worker/v1/jobs/%s", token), `{}`, http.StatusNotFound, `*`)
 }
 
+func TestArgs(t *testing.T) {
+	distroStruct := fedoratest.New()
+	arch, err := distroStruct.GetArch("x86_64")
+	require.NoError(t, err)
+	imageType, err := arch.GetImageType("qcow2")
+	require.NoError(t, err)
+	manifest, err := imageType.Manifest(nil, distro.ImageOptions{Size: imageType.Size(0)}, nil, nil, nil, 0)
+	require.NoError(t, err)
+
+	tempdir, err := ioutil.TempDir("", "worker-tests-")
+	require.NoError(t, err)
+	defer os.RemoveAll(tempdir)
+	server := newTestServer(t, tempdir)
+	server.Handler()
```

Comment:
This is a no-op. Maybe we want to actually use `test.TestRoute` to test the also that the API works as expected?

Reply:
Thanks for catching that.  Probably missed that it's not necessary when copying from other tests.

There's no API endpoint for getting args from the **worker**.  It just grabs it directly from the `fsJobQueue`.  The `newTestServer()` is used to initialise the job queue to test the worker methods that wrap the new `JobQueue` methods.

Reply:
Right, sorry!